backup/setup/roles.json

```diff
@@ -142,8 +144,7 @@
 			"SCHOOL_SYSTEM_EDIT",
 			"SCHOOL_SYSTEM_VIEW",
 			"USER_CHANGE_OWN_NAME",
-			"MEDIA_SCHOOL_LICENSE_ADMIN",
-			"SCHOOL_LIST_DISCOVERABLE_TEACHERS"
+			"MEDIA_SCHOOL_LICENSE_ADMIN"
```

Comment:
depending on the changed order, this should go in line 108 right?

Reply:
My change was, that I removed SCHOOL-LIST-Permission, as it was already at it was a duplicate and already in line 115. Also moved MEDIA_SCHOOL-Permission to it's correct position - as you suggested.